EIPS/eip-4672.md

```diff
@@ -0,0 +1,56 @@
+---
+eip: 4672
```

Comment:
```suggestion
eip: 4742
```

@abcoathup is correct (https://github.com/ethereum/EIPs/pull/4742#issuecomment-1029810697), this is used the wrong number.

Reply:
@0xfoobar can you update the title of the PR to use EIP4742 as well as update the filename.

Reply:
Done